sourced/ml/models/id_splitter.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import os
+import string
+
+import keras
+from keras import backend
+from keras.preprocessing.sequence import pad_sequences
+from modelforge import Model, register_model
+import numpy as np
+# import tensorflow as tf
+
+from sourced.ml.algorithms.id_splitter.nn_model import (f1score, precision,
+                                                        recall)
+from sourced.ml.models.license import DEFAULT_LICENSE
+
+MAXLEN = 40
```

Comment:
I think `MAXLEN` is model dependent and we should include it to the model.

Reply:
Included in the model, this is just the default value.

Reply:
Still it's better to put it inside the class, like in `TokenParser`. We do it like that everywhere, if it's not shared among several classes or functions.